packages/react-server/src/ReactFizzResources.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
I'm not sure what this indirection represents. `ReactFizzServer` is the semi-public API that implementors of a renderer can interact with.

It's not clear to me what these other indirections do and what is public vs private.

It seems like they can all go into ReactFizzServer and then we choose there what is private and not.